python-avd/tests/pyavd/j2filters/test_natural_sort.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+import pytest
+from pyavd.j2filters.natural_sort import convert, natural_sort
+
+
+class TestNaturalSortFilter:
+    @pytest.mark.parametrize("item_to_convert", ["100", "200", "ABC"])
```

Comment:
please correct me if am wrong, convert function accept str but here we are passing list?